docs/reference/index-modules/slowlog.asciidoc

```diff
@@ -83,6 +83,36 @@ logger.index_search_slowlog_rolling.appenderRef.index_search_slowlog_rolling.ref
 logger.index_search_slowlog_rolling.additivity = false
 --------------------------------------------------
 
+==== Identifying search slow log origin
+
+It is often useful to identify what triggered a slow running query. If a call was initiated with an `X-Opaque-ID` header, then this value
+will be present in Search Slow logs as an additional **id** field (scroll to the right).
+```
```

Comment:
I'd rewrite this to use the Asciidoc source code blocks.

For example:
```
[source,txt]
----
[2019-08-30T11:59:37,786][WARN ][i.s.s.query              ] [node-0] [index6][0] took[78.4micros], took_millis[0], total_hits[0 hits], stats[], search_type[QUERY_THEN_FETCH], total_shards[1], source[{"query":{"match_all":{"boost":1.0}}}], id[MY_USER_ID],
----
```

Also consider replacing `[2019-08-30T11:59:37,786]` with a more future-proof timestamp like `[2030-08-30T11:59:37,786]`